src/pages/Contact.css

```diff
@@ -0,0 +1,138 @@
+.titleContainer{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+
+}
+.title{
+    /* style the CONTACT US */
+    width: 468px;
+    height: 68px;
+    /* margin-left: 468px; */
+    margin-left: 242px;
+    top: 159px;
+    
+
+    font: PingFang SC;
+    font-weight: 500;
+    font-style: normal;
+    font-size: 48px;
+    /* text-align: center;  */
+}
+
+.bottomContainer{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+
+}
+.messageContainer{
+    display: flex;
+    flex-direction: column;
+
+    /* why isn't this working? */
+    align-content: space-between;
+
```

Comment:
Add a `height: ...` to the container, and change `justify-content: center` to `justify-content: space-between` while making `align-items: center`